src_c/mixer.c

```diff
@@ -760,6 +760,22 @@ snd_get_raw(PyObject *self, PyObject *_null)
                                      (Py_ssize_t)chunk->alen);
 }
 
+static PyObject *
+snd_copy(PyObject *self, PyObject *_null)
+{
+    Mix_Chunk *old_chunk = pgSound_AsChunk(self);
+    Mix_Chunk *new_chunk = Mix_QuickLoad_RAW(old_chunk->abuf, old_chunk->alen);
+    new_chunk->volume = old_chunk->volume;
+    pgSoundObject *sound =
+        (pgSoundObject *)pgSound_Type.tp_new(Py_TYPE(self), NULL, NULL);
```

Comment:
I'd change this to something like `Py_TYPE(self)->tp_new(Py_TYPE(self), NULL, NULL);` so that the `__new__` of the subclass is used (if overridden)

Additionally, I'd like if the `Mix_QuickLoad_RAW` came *after* a successful `tp_new` call, and complete with error handling for when it fails